autogen/coding/local_commandline_code_executor.py

```diff
@@ -161,7 +161,8 @@ def execute_code_blocks(self, code_blocks: List[CodeBlock]) -> CommandLineCodeRe
                 break
 
         code_file = str(file_names[0]) if len(file_names) > 0 else None
-        return CommandLineCodeResult(exit_code=exitcode, output=logs_all, code_file=code_file)
+        
+        return CommandLineCodeResult(exit_code=exitcode, output=logs_all, code_file=code_file, max_output_length=2000)
```

Comment:
Rather than adding logic to the return data object to truncate the length, can you please add an argument to the constructor which defaults to no truncation and truncate `logs_all` before creating the `CommandLineCodeResult`

Reply:
Yes thank you for the sudgestion I would like to make summarization of the error, providing a truncated version of the error is not the best (open to sudgestion)